src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/SendPolicyRequirement.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Bit.Core.AdminConsole.Enums;
+using Bit.Core.AdminConsole.Models.Data.Organizations.Policies;
+using Bit.Core.Models.Data.Organizations.OrganizationUsers;
+
+namespace Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements;
+
+public class SendPolicyRequirement : IPolicyRequirement
+{
+    public bool DisableSend { get; init; }
```

Comment:
```suggestion
    public bool DisableSend { get; private init; }
```

Reply:
I've moved the PolicyRequirement implementations to a separate branch but I'll make a note of it 👍